examples/nad_with_syncer.yaml

```diff
@@ -0,0 +1,27 @@
+# WARNING: This is an example definition only. Remove all comments before use.
+
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: afxdp-network                                        # Name of this network, pods will request this network by name
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: afxdp/myPool            # Needs to match the device plugin pool name / resource type
+spec:
+  config: '{
+      "cniVersion": "0.3.0",
+      "type": "afxdp",                                       # CNI binary, leave as afxdp
+      "mode": "primary",                                     # CNI mode setting (required)
+      "logFile": "afxdp-cni.log",                            # CNI log file location (optional)
+      "logLevel": "debug",                                   # CNI logging level (optional)
+      "dpSyncer": true,                                      # Sync with Device Plugin over gRPC (optional)
```

Comment:
should make this more explicit that it's only for bpfmap pinning or on the DP side if pinning isn't enabled then just return success...